libraries/AP_DDS/AP_DDS_Client.h

```diff
@@ -179,6 +179,9 @@ class AP_DDS_Client
     //! @brief Update the internally stored DDS messages with latest data
     void update();
 
+    //! @brief GCS message prefix
+    static constexpr char* msg_prefix{"DDS:"}; 
```

Comment:
```suggestion
    static constexpr char* msg_prefix = "DDS:"; 
```

.. may also need to be `const` rather than `constexpr`... but I doubt that.

Reply:
Ah, `clang` let that through in local build. Rebased and changed.